WebService/v1/Controllers/SimulationsController.cs

```diff
@@ -81,6 +88,25 @@ public class SimulationsController : Controller
                 await this.simulationsService.UpsertAsync(simulation.ToServiceModel(id)));
         }
 
+        [HttpPut("{id}/DeleteDevices")]
```

Comment:
`DeleteDevices` is a custom verb based on PUT. we may use the recent API endpoint standard. e.g. !DeleteDevices?id={id}. Here is an example in Devis's PR FYI: https://github.com/Azure/pcs-template-microservice-dotnet/blob/0a72e604f5b6e544ba943f02ff966b7ab7930436/WebService/v1/Controllers/ValuesController.cs#L105

Reply:
For this API, we want to pass a list of device ids